### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by dustman9000**
Add myself to reviewers and approvers.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -4,11 +4,13 @@ reviewers:
 - dofinn
 - mbarnes
 - yithian
+- dustman9000
 approvers:
 - c-e-brumm
 - cblecker
 - dofinn
 - mbarnes
```
Comment (cblecker) [inline OWNERS:12]: Would you mind removing Matt while you're in here?

Comment (dustman9000) [reply OWNERS]: done

**Pull request merged**